metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
Sorry, I thought about it a bit more and I'm not very happy with the outcome here. It's mostly my fault, too. 

Towards the beginning of the thread I thought we were talking about values returned by the systems. So like if Redis returns its average response time, we would put that in the `response_time` field, which would be a common field for all systems. Halfway through I realized it's about the queries that Metricbeat itself creates, and how long those take.

In this case, I think a more standard name is `round_trip_time` or `rtt`, because it doesn't only measure the response time, but the response time + the network delay between Metricbeat and the system. I'd prefer having a different name then what Packetbeat uses, because they really are different things. What do you think?

I think we can leave it always enabled and people can remove it with generic filtering (potentially conditionally if it's bigger than a threshold).